lib/util.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+
+exports.starterPlan = a => !!a.plan.name.match(/(dev|basic)$/)
+
+exports.configVarNamesFromValue = (config, value) => {
+  const sortBy = require('lodash.sortby')
+
+  let keys = []
+  for (let key of Object.keys(config)) {
+    if (config[key] === value) keys.push(key)
+  }
+  return sortBy(keys, k => k !== 'DATABASE_URL', 'name')
+}
+
+exports.databaseNameFromUrl = (uri, config) => {
+  const url = require('url')
+
+  let names = exports.configVarNamesFromValue(config, uri)
+  let name = names.pop()
+  while (name === 'DATABASE_URL') name = names.pop()
```

Comment:
`> 1`, right? The condition is evaluated before the loop body, so you want to make sure you have something to pop off.